client/src/Components/CreateOffer/index.jsx

```diff
@@ -1,11 +1,186 @@
+    // const member_id = get member_id from local storage
+    // this.setState({ member_id })
+  }
+
+  handleSubmit = () => {
+    // eslint-disable-next-line react/prop-types
+    // const { history } = this.props;
+    const { title, position, description, offerType } = this.state;
+
+    newOfferValidation
+      .validate(
+        { title, position, description, offerType },
+        { abortEarly: false }
+      )
+      .then(() => {
+        this.setState({ errMsg: '' });
+        // send request to the backend with body
+        // axios
+        // .post('/api/v1/offers', { title, position, description, skills, offerType })
+        // get offer_id from response
+        // history.push(`/app/${offer_id}`);
+        // send request with new skills from memeber (which don't exist in autocomplete) to update filters table `PUT` : `/api/v1/filter/:memberId`
+        // and the same for offerTypes
+        // body of the request { skills: this.state.skills , offerTypes: this.state.offerTypes}
+      })
+      .catch(({ inner }) => {
+        if (inner) {
+          const errors = inner.reduce(
+            (acc, item) => ({ ...acc, [item.path]: item.message }),
+            {}
+          );
+          this.setState({ errMsg: { ...errors } });
+        }
+      });
+  };
+
+  handleTitle = title => this.setState({ title });
+
+  handlePosition = position => this.setState({ position });
+
+  handleDescription = description => this.setState({ description });
+
+  handleSkills = skills => this.setState({ skills });
+
+  handleOfferTypes = offerType => this.setState({ offerType });
+
+  render() {
+    // eslint-disable-next-line react/prop-types
+    const { history } = this.props;
+    const { errMsg } = this.state;
+    return (
+      <Container className="page__container newoffer__container">
+        <h1 className="newoffer__title"> New Offer </h1>
+        <Form>
+          <Form.Group>
+            <div className="newoffer__content">
+              <Row>
+                <Col sm={12} md={12} lg={6}>
+                  <Form.Label className="newoffer__label"> Title: </Form.Label>
+                  <Form.Control
+                    type="text"
+                    placeholder="eg:  Small Shop seeks workers "
+                    onChange={({ target }) => this.handleTitle(target.value)}
+                  />
+                  {errMsg.title && (
+                    <div className="newoffer__errMsg">
+                      <i className="fas fa-exclamation newoffer__errMsg--icon" />{' '}
+                      {errMsg.title}
+                    </div>
+                  )}
+                  <Form.Label className="newoffer__label">
+                    {' '}
+                    Position:{' '}
+                  </Form.Label>
+                  <Form.Control
+                    type="text"
+                    placeholder="eg:   Marketing Manager"
+                    onChange={({ target }) => this.handlePosition(target.value)}
```

Comment:
this way you are using an arrow function here also another function `this.handlePosition` which is not necessary